common/csp-directives.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+const { concat } = require('lodash');
+
+module.exports = function cspDirectives({
+    enableHotjar = false,
+    allowLocalhost = false
+} = {}) {
+    let defaultSrc = [
+        "'self'",
+        '*.biglotteryfund.org.uk',
+        '*.tnlcommunityfund.org.uk',
+        '*.google.com',
+        '*.facebook.com',
+        '*.twitter.com',
+        '*.gstatic.com',
+        '*.twimg.com',
+        '*.youtube.com',
+        '*.vimeo.com',
+        'cdn.polyfill.io',
+        'cdn.syndication.twimg.com',
+        'maxcdn.bootstrapcdn.com',
+        'ajax.googleapis.com',
+        'cdnjs.cloudflare.com',
+        'platform.twitter.com',
+        'sentry.io',
+        'syndication.twitter.com',
+        'www.google-analytics.com',
+        'use.typekit.net'
+    ];
+
+    if (allowLocalhost) {
```

Comment:
This was previously two blocks of code: one for local images,  and another for browser-stack, I've renamed this to be an `allowLocalhost` feature flag as this more directly describes what we are doing here.